src/stylesheets/components/_components.swiper.scss

```diff
@@ -0,0 +1,75 @@
+//overwrite default swiper styling by iDangerous
+
+$swiper-width: 236px;
+$swiper-height: 180px;
+$swiper-height-desktop: 143px;
+
+.swiper-container {
+  width: $swiper-width;
+  height: $swiper-height;
+}
+
+.swiper-pagination {
+  bottom: 0;
+  left: 85px;
+  display: flex;
+  justify-content: space-between;
+  width: 56px;
+}
+
+.swiper-container-horizontal > .swiper-pagination-bullets {
+  bottom: -6px;
+}
+
+.swiper-pagination-bullet {
+  background-color: $celery;
+  opacity: 0.5;
+}
+
+.swiper-pagination-bullet-active {
+  background-color: $celery;
+  opacity: 1;
+}
+
+.c-swiper {
+  background-color: $athens-gray;
+  padding: 64px 0 400px 0;
+  background-image: url("../images/swiper.png");
+  background-size: 275% 85%;
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  // display: flex;
+  // flex-direction: column;
+  // align-items: center;
+  @include mq ($from: tablet) {
+    background-position: center;
+    background-size: cover;
+    padding: 96px 0 493px 0;
+  }
+}
+
+.c-swiper__header {
+  text-align: center;
+  h1 {
+    color: $black;
+    font-size: 20px;
+  }
+}
+
+.c-swiper__content {
+  @include mq($from:tablet) {
+    width: $swiper-width;
+    height: $swiper-height-desktop;
+  }
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  p {
```

Comment:
this may want to just apply to the `c-swiper__content` block, because if we change from a `p` to a `h5` or some other block element then the text won't be centred

Reply:
👍